src/pages/Community/FeedDetail/ReReply.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+import "./ReReply.scss";
+class ReReply extends Component {
```

Comment:
```suggestion
import React, { Component } from "react";
import "./ReReply.scss";

class ReReply extends Component {
```
구분되는 내용끼리는 한줄 띄워주세요!

Reply:
넵 수정하겠습니다!!